.github/workflows/codeql-analysis.yml

```diff
@@ -36,13 +36,13 @@ jobs:
     #    and modify them (or add more) to build your code if your project
     #    uses a compiled language
 
-    - name: Setup .NET 5.0.408
+    - name: Setup .NET 6.0.301
       uses: actions/setup-dotnet@v2.1.0
       with:
-        dotnet-version: 5.0.408
+        dotnet-version: 6.0.301
```

Comment:
can we use 6.x ?

Reply:
yes, but sometimes it's not happy with a wildcard